### PR TITLE
Add tmux-cssh as replacement for clusterssh

### PR DESCRIPTION
**Pull request by vmcj**
I can't find if there is an extra config but that can be easily added if @tuupke / @meisterT has such a thing.

### CODE REVIEW

**Comment by meisterT** [comment]
Yeah, I had some bash aliases, basically:
```
alias ssh-jh='tmux-cssh domjudge-judgehost1 domjudge-judgehost2 ...'
alias ssh-servers='tmux-cssh domserver-primary domserver-backup ...'
alias ssh-all=...
alias ssh-analyst=...
```

We can also work on this onsite

**Comment by vmcj** [comment]
> Yeah, I had some bash aliases, basically:
> 
> ```
> alias ssh-jh='tmux-cssh domjudge-judgehost1 domjudge-judgehost2 ...'
> alias ssh-servers='tmux-cssh domserver-primary domserver-backup ...'
> alias ssh-all=...
> alias ssh-analyst=...
> ```
> 
> We can also work on this onsite

I've added something like that now, went with shell scripts as that was easier.

File: provision-contest/ansible/roles/clusterssh/tasks/main.yml [hunk 1]
```diff
@@ -22,3 +22,46 @@
...
+
+- name: Set tmux-cssh shorthands
+  template:
+    owner: root
+    group: root
+    mode: 0755
+    src: tmux-cluster.sh.j2
+    dest: /usr/local/bin/{{ item.short }}
+  loop:
+    - short: ssh-js
```
Comment (meisterT) [inline provision-contest/ansible/roles/clusterssh/tasks/main.yml:52]: I prefer `jh` but if it is just me I can alias it 😀

**Review by meisterT — APPROVED** [review]

File: provision-contest/ansible/roles/clusterssh/tasks/main.yml (hunk 1 above)
Comment (vmcj) [reply provision-contest/ansible/roles/clusterssh/tasks/main.yml]: I'm fine with any comment, so feel free to push the right naming afterwards. I already changed the `js` to `jh` as it does indeed make more sense.

For the group take a look at the hosts file where subgroups do matter (so the group `domserver` does contain everything in the group and the hosts from its children)

**Review by Kevinjil — APPROVED** [review]

**Pull request merged**